### DISCUSSION TITLE
iOS Mobile Hitchwiki

### DISCUSSION THREAD

**Discussion by CichoszKacper** [Hitchwiki/hitchwiki#213] [Ideas]
Hi, I have an idea to create a downloadable version of Hitchwiki.org for iOS. Users would be able to download the latest version of data dumps and display them in the form of a mobile app. I tried using your database dumps and transforming the data into a mobile-readable product. However, this turned out to be a very tedious task, as the XML is not very friendly. I was wondering if there is any chance we could collaborate on this. Any more readable API would be groundbreaking for me!

**Comment by guaka** [comment]
Cool.

This seems relevant: https://github.com/search?q=org%3AHitchwiki+kiwix&type=issues

Parsoid could be useful for this. It could also be nice to get Hitchwiki available in Kiwix. Have you looked at this?

I think https://hitchwiki.org/en/Hitchwiki:While_offline could use some edits to update the current state of aard, kiwix, etc.

**Reply by CichoszKacper** [reply]
Thanks for the reply. I will have a look into Parsoid. I can't seem to download the latest database dumps. You have a small disclaimer saying that there is some problem with them. Is there some other way that I can access them for testing?
